doc/pi_netrw.jax

```diff
@@ -1145,35 +1145,40 @@ QUICK HELP						*netrw-quickhelp* {{{2
 ブックマークは $HOME/.netrwbook に保存されます。ブックマークは常にソートされま
 す。
 
-If there are marked files and/or directories, mb will add them to the bookmark
-list.
+もしマークされたファイル/ディレクトリが存在すれば、 mb はそれらをブックマーク
+に追加します。
 
 *netrw-:NetrwMB*
-Addtionally, one may use :NetrwMB to bookmark files or directories. >
+また、ファイルかディレクトリをブックマークするのに :NetrwMB も使用できます。 >
 
 	:NetrwMB[!] [files/directories]
 
-< No bang: enters files/directories into Netrw's bookmark system
+<bangが付かない場合: ファイル/ディレクトリを
+                     Netrwのブックマークシステムに追加します。
 
-   No argument and in netrw buffer:
-     if there are marked files        : bookmark marked files
-     otherwise                        : bookmark file/directory under cursor
-   No argument and not in netrw buffer: bookmarks current open file
-   Has arguments                      : |glob()|s each arg and bookmarks them
+   netrw バッファで引数なし:
+     もしマークされたファイルが存在する: マークされたファイルを
+                                         ブックマークする
+     その他                            : カーソル位置のファイル/
```

Comment:
要修正: `otherwise` = `そうでなければ`

前行に `if there are marked files` とあるので、ここの `otherwise` は `else` すなわち
`if there are not any marked files` の意味になる。
`その他` ではまったく意味が不明。